database/migrations/2025_11_26_081715_add_comment_to_matchmaking_pools.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('matchmaking_pools', function (Blueprint $table) {
+            $table->string('comment')->nullable()->after('name');
```

Comment:
Probably call it `description` because I think it'd be useful to show this as tooltip or subtext.

(and maybe preemptively include in api response while we're here)